core/bin/zksync_api/src/fee_ticker/tests.rs

```diff
@@ -581,7 +582,7 @@ async fn test_error_coingecko_api() {
         Default::default(),
         FakeTokenWatcher,
     );
-    let connection_pool = ConnectionPool::new(Some(1));
+    let connection_pool: ConnectionPool = ConnectionPool::new(Some(1));
```

Comment:
```suggestion
    let connection_pool = ConnectionPool::new(Some(1));
```
Should we declare type or let Rust infer automatically.

Reply:
Changed.